docs/releases/cloud/point-2025-11-11-release-note.md

```diff
@@ -0,0 +1,38 @@
+---
+description: This article lists improvements and bug fixes in the Syskit Point Cloud version 2025.5.117.30
+---
+
+# November 11, 2025
+
+[Start a free trial](https://www.syskit.com/products/point/free-trial/) and [tell us what you think](https://www.syskit.com/company/contact-us/).
+
+## About Syskit Point Cloud
+
+* **New to Syskit Point Cloud?** Explore Syskit Point Cloud with a 21-day [free trial](https://www.syskit.com/products/point/free-trial/) for an easy and effective way to manage and secure your environment.
+
+* **Already using Syskit Point Cloud?** Syskit Point Cloud is automatically upgraded to the latest version when available. The automatic update occurs outside working hours to ensure minimal interference with your day-to-day business. The new version will begin rolling out with this announcement and is expected to reach all customers within the next few days.
+
+## Improvements & Bug Fixes 
+
+* **Improvements made to Provisioning.**
+  * An option was added when creating a Provisioning template that allows you to limit the maximum number of characters allowed in workspace URLs.
+    * When this option is enabled, any URL exceeding the limit will be automatically shortened (? tako sam skuzila bar?). 
```

Comment:
da, ali će se i prikazati konačni URL, tako da je end-user svjestan što se dešava i kako će to na kraju izgledati